paasta_tools/utils.py

```diff
@@ -294,6 +294,10 @@ def get_pool(self):
         :returns: the "pool" attribute in your config dict, or the string "default" if not specified."""
         return self.config_dict.get('pool', 'default')
 
+    def get_pool_constraints(self):
+        pool = self.get_pool()
+        return [["pool", "LIKE", pool]]
```

Comment:
I'm not sure this is a good invocation of DRY -- marathon needs LIKE or CLUSTER (maybe?), chronos can use LIKE or EQUALS. It's somewhat coincidental that they both have similar constraint languages, and future frameworks might not have overlapping constraint languages.

It's probably good to unify on LIKE for consistency, though.